backend.py

```diff
@@ -280,41 +281,50 @@ def get_active_robots(self):
                 yield robot
 
 
-def get_start_tiles(board):
+def get_robot_names():
+    """
+    Return a list of robots names using pathlib.Path library.
     """
-    Get start tiles for robots.
+    robot_names = []
+    for img in Path('./img/robots/png').iterdir():
+        robot_name = img.stem
+        robot_names.append(robot_name)
+    return robot_names
+
+
+def get_start_tiles(board, robot_tile_type):
```

Comment:
A co udělat `robot_tile_type` jako volitelný argument? Např. `get_start_tiles(board, robot_tile_type=None)`, abychom nemusely měnit volání té funkce jinde v kódu. Ikdyž jsi to už udělala... Je to jenom návrh.

Reply:
Jo, premyslela jsem o tom! Dokazes zajistit, aby ten volitelny argument se pridal jen v `get_start_tiles`, ale v pripade, ze budu chtit vytvorit stav hry se "stop" hodnotou, ta se korektne predala dal? Protoze to mi puvodne failovalo, musela jsem zajistit predani teto hodnoty v retezci volani: `get_start_state` -> `create_robots` -> `get_start_tiles`

Reply:
Jo takhle... Popřemýšlím nad tím.

Reply:
Možná jsou to blbé otázky, ale snažím se pochopit :) Potřebuješ vytvářet stop stav úplně stejně jako start stav? Co ze stop stavu budeš vlastně potřebovat? Jenom pozice stop políček? Nebo budeš tyto dva stavy nějak porovnávat?